Scripts/parameters/demand/wb_business.json

```diff
@@ -165,47 +163,47 @@
     },
     "mode_choice": {
         "car_work": {
-            "constant": -0.6693,
+            "constant": -2.25056,
             "generation": {},
             "attraction": {},
             "impedance": {},
             "log": {
-                "logsum": 0.9598
+                "logsum": 1
```

Comment:
Should the model structure be turned around?

Reply:
We're probably trying to combine this group with other as number of observations is low in near future. Therefore did only simple constraint here.